t/08-autotest.t

```diff
@@ -486,6 +486,22 @@ subtest 'start_process' => sub {
     stderr_like { $process->{code}->(); } qr/Snapshots are not supported/, 'run_all outputs status on stderr';
 };
 
+subtest 'lua_use' => sub {
+    my $lua_vars = {};
+    $mock_autotest->redefine('lua_set' => sub ($k, $v) { $lua_vars->{$k} = $v; });
+    autotest::_lua_use('testapi');
+    is $lua_vars->{realname}, 'Bernhard M. Wiedemann', 'Check importing strings';
+    is ref($lua_vars->{assert_script_run}), 'CODE', 'Check importing functions';
+
+    autotest::_lua_use('testlib');
+    is $lua_vars->{testfunc1}(), 42, 'Exported function is imported';
+    is $lua_vars->{testfunc2}, undef;
+    autotest::_lua_use('testlib', ['testfunc2']);
+    is $lua_vars->{testfunc2}(), 43, 'Explicit imports of non-exported functions';
+    is_deeply $lua_vars->{testarray}, [1, 2, 3], 'Import Array';
+    is_deeply $lua_vars->{testhash}, {foo => 'bar'}, 'Import Hash';
+};
```

Comment:
I ran the tests and noticed that the `pre_boot_lua` module is never tested.

```
$autotest::tests{'tests-pre_boot_lua'}->runtests
```
seems to work basically, if I comment out the `check_screen` and `send_key`.

Reply:
Okay - the `08-autotest.t` test should now run a lua module.